docs/editing_guidelines.md

```diff
@@ -90,6 +90,25 @@ If the main source for an assertion is a term in another ontology, the short ide
 
 ORCID identifiers may also be used when the only available source for an assertion is an individual researcher. This should be done sparingly.
 
+**Technical details of adding a cross-reference using Protégé**:
+
+For all: In the "Create Annotation" window, select the annotation property **database_cross_reference**.
+
+If the cross-reference can be represented as a CURIE and is not an ORCID: Enter the CURIE, using the bioregistry standard prefix, as a Value on the "Literal" tab. Leave Datatype empty.
```

Comment:
```suggestion
If the cross-reference can be represented as a CURIE and is not an ORCID: Enter the CURIE, using the [bioregistry OBO context](https://bioregistry.io/context/obo) prefix ([link to prefixmap](https://github.com/biopragmatics/bioregistry/blob/main/exports/contexts/obo.context.jsonld), as a Value on the "Literal" tab. Leave Datatype empty.
```

Reply:
Of note, the “bioregistry OBO context” seems to recommend using `DOI` instead of `doi` as the “prefix” for DOI identifiers, which is against both what is recommended by the DOI specification and what we recommend ourselves slightly earlier in this very document.

Reply:
Additionally, this is against bioregistry itself - it is a manual override done by the OBO context because most ontologies refer to dois using capital prefix. I am not opposed to trying to make a change here, but I would say we should stick to the context and debate capitalisation in that context rather than here

Reply:
For the purpose of this documentation, should the advise be to follow the [bioregistry standard](https://bioregistry.io/registry/doi) (doi) or the [OBO context](https://bioregistry.io/context/obo) (DOI)?

Reply:
I cant make that call alone, but I would prefer OBO context.

Reply:
Added to CL meeting agenda

Reply:
I think the people at your meeting wont be able to answer that question; its better to ask @anitacaron to raise it at the OBO ontology tech call (which as it so happens is also today), because this should not be decided on an ontology by ontology level.

Reply:
> I think the people at your meeting wont be able to answer that question; its better to ask @anitacaron to raise it at the OBO ontology tech call (which as it so happens is also today), because this should not be decided on an ontology by ontology level.

@anitacaron, can you raise this issue and the [one on URL links below](https://github.com/obophenotype/cell-ontology/pull/2058#discussion_r1256022366) at the OBO onto tech call?

Reply:
Sure

Reply:
Per @anitacaron, it was decided that `doi:` should be the standard.